quantity.hpp

```diff
@@ -361,9 +600,9 @@ class quantity {
 	template<typename U, typename S, typename V, typename T, typename E = value_type>
 	constexpr bool is_near(const quantity<U,S,V,T>& rhs, const E epsilon = get_epsilon<V>()) const
 	                                                                      {
-		                                                                      static_assert( are_units_compatible<unit_type , U>::value, "fundamentally incompatible units");
-		                                                                      static_assert( std::is_same        <value_type, V>::value, "different value types (use value_cast)");
-		                                                                      static_assert( std::is_same        <tag_type  , T>::value, "incompatible tags (use tag_cast)"      );
+		                                                                      static_assert(are_units_compatible_v<unit_type , U>, "fundamentally incompatible units"      );
+		                                                                      static_assert(detail::is_same_v     <value_type, V>, "different value types (use value_cast)");
+		                                                                      static_assert(are_tags_compatible_v <tag_type  , T>, "incompatible tags (use tag_cast)"      );
```

Comment:
again here, substraction needs `is_same_v<tag_type, T>`

Reply:
I am not so sure here ("is active power near reactive power?"), but in absence of experience with this feature I'm happy to be more restrictive until this is shown to be too inconvenient.